src/Symfony/Component/Console/Tester/ApplicationTester.php

```diff
@@ -31,16 +33,23 @@ class ApplicationTester
 {
     private $application;
     private $input;
+
+    /**
+     * @var OutputInterface|ConsoleOutputInterface
+     */
     private $output;
+    private $captureOutputStreamsIndependent;
 
     /**
      * Constructor.
      *
-     * @param Application $application An Application instance to test.
+     * @param Application $application                     Application to test.
+     * @param bool        $captureOutputStreamsIndependent
      */
-    public function __construct(Application $application)
+    public function __construct(Application $application, $captureOutputStreamsIndependent = false)
```

Comment:
Instead of adding a constructor argument, I would instead add an option to the `run()` method.